src/Symfony/Bridge/Doctrine/Form/Type/DoctrineType.php

```diff
@@ -288,13 +258,11 @@ public function configureOptions(OptionsResolver $resolver)
 
         $resolver->setDefaults(array(
             'em' => null,
-            'property' => null, // deprecated, use "choice_label"
             'query_builder' => null,
-            'loader' => null, // deprecated, use "choice_loader"
             'choices' => null,
             'choices_as_values' => true,
             'choice_loader' => $choiceLoader,
-            'choice_label' => $choiceLabel,
+            'choice_label' => array(__CLASS__, 'createChoiceLabel'),
```

Comment:
I'm not 100% sure about this change, because of this comment above it in the default closure: `// BC: use __toString() by default`

However, I'm unsure what to use as a default value otherwise.